src/ScottPlot4/ScottPlot/Plottable/Image.cs

```diff
@@ -51,10 +56,32 @@ public class Image : IPlottable
         public int XAxisIndex { get; set; } = 0;
         public int YAxisIndex { get; set; } = 0;
 
-        public override string ToString() => $"PlottableImage Size(\"{Bitmap.Size}\") at ({X}, {Y})";
-        public AxisLimits GetAxisLimits() => new(X, X, Y, Y);
         public LegendItem[] GetLegendItems() => Array.Empty<LegendItem>();
 
+        public override string ToString()
+        {
+            if (SizeInAxisUnits is SizeF axisSize)
+            {
+                return $"PlottableImage Axis Size(\"{axisSize}\") at ({X}, {Y})";
+            }
+            else
+            {
+                return $"PlottableImage Size(\"{Bitmap.Size}\") at ({X}, {Y})";
+            }
+        }
+
+        public AxisLimits GetAxisLimits()
+        {
+            if (SizeInAxisUnits is SizeF axisSize)
```

Comment:
I didn't know that the `is` operator to did a null check when the right-hand operand isn't `null`, very cool.

My first guess was that this worked because this was casting from `SizeF?` to `SizeF`, but I checked and it works with reference types as well (with nullable reference types disabled), meaning it's not a cast.